src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +43,41 @@
      * Андрей - 600
      * София - 900</p>
      */
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+        LocalDate fromDate = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate toDate = LocalDate.parse(dateTo, FORMATTER);
+        StringBuilder report = new StringBuilder();
+
+        if (toDate.isBefore(fromDate)) {
+            throw new IllegalDateParametersException("Wrong parameters");
+        }
+
+        report.append("Отчёт за период " + dateFrom + " - " + dateTo);
+
+        for (String name : names) {
+            report.append("\n" + name + " - ");
+            int totalPersonSalary = 0;
+            for (String elementOfData : data) {
+                if (elementOfData.split(" ")[1].equals(name)
```

Comment:
I can't see any changes

Reply:
How it`s possible? I did it yesterday and I see it now. What should I do?
int totalPersonSalary = 0;
            for (String elementOfData : data) {
                String [] splittedPartOfData = elementOfData.split(" ");
                if (splittedPartOfData[1].equals(name)